boot/zephyr/Kconfig

```diff
@@ -202,7 +202,7 @@ config BOOT_SIGNATURE_TYPE_PURE_ALLOW
 
 choice BOOT_SIGNATURE_TYPE
 	prompt "Signature type"
-	default BOOT_SIGNATURE_TYPE_ED25519 if BOARD_NRF54L15PDK_NRF54L15_CPUAPP
+	default BOOT_SIGNATURE_TYPE_ED25519 if SOC_NRF54L15_CPUAPP
```

Comment:
this part needs to be a fixup of https://github.com/nrfconnect/sdk-mcuboot/commit/3cdcdb32f254248e6d2cd87dce09597723782562 in separate commit